discovery-provider/src/models/notifications/notification.py

```diff
@@ -40,3 +41,16 @@ class NotificationSeen(Base, RepresentableMixin):
     txhash = Column(String)
     seen_at = Column(DateTime, nullable=False)
     PrimaryKeyConstraint(user_id, seen_at)
+
+
+class PlaylistSeen(Base, RepresentableMixin):
+    __tablename__ = "playlist_seen"
+
+    is_current = Column(Boolean, nullable=False)
```

Comment:
how come the other models don't need is_current?

Reply:
So for playlist seen, we only care about the most recent one.
Ideally we would have a single row for a user to playlist and update that timestamp, but adding is_current fits better with the model. 
One reason notifications doesn't have is current is because it's created off triggers and no past notification is invalidated. But note that it does not handle reverts.